README.md

```diff
@@ -244,4 +244,12 @@ rm -rf new_venv
 
 ## VS Code <a name="vscode"></a>
 
+### Extensions
+
+- [Git graph](https://marketplace.visualstudio.com/items?itemName=mhutchie.git-graph)
+- [Juptyer](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
```

Comment:
[Juptyer] -> [Jupyter]

Reply:
Thanks. I (or someone else?) will correct the mistake in separately